Makefile

```diff
@@ -32,7 +32,7 @@ occ=$(CURDIR)/../../occ
 private_key=$(HOME)/.owncloud/certificates/$(app_name).key
 certificate=$(HOME)/.owncloud/certificates/$(app_name).crt
 sign=$(occ) integrity:sign-app --privateKey="$(private_key)" --certificate="$(certificate)"
-sign_skip_msg="Skipping signing, either no key and certificate found in $(private_key) and $(certificate) or occ can not be found at $(occ)"
+sign_skip_msg="Skipping signing, either no key (.key) and certificate (.crt) found in $(private_key) and $(certificate) or occ can not be found at $(occ)"
```

Comment:
I am running 
```
$ openssl req -nodes -newkey rsa:4096 -keyout /Users/mrow4a/.owncloud/certificates/files_paperhive.key -out /Users/mrow4a/.owncloud/certificates/files_paperhive.crt -subj "/CN=files_paperhive"
$ make dist
```
and still getting `App signature is invalid: App Certificate is not valid.`
any clue @DeepDiver1975 ?

Reply:
did you ever sign your csr properly as documented?

Reply:
@DeepDiver1975 https://github.com/owncloud/appstore-issues/issues/137